ui/src/components/MainContainer.js

```diff
@@ -7,6 +7,7 @@ import { MuiPickersUtilsProvider } from "material-ui-pickers";
 import SermonContainer from "./sermons/SermonContainer";
```

Comment:
assuming all UI changes below will be modified again and not final

Reply:
yup that's temporary only